author-avatars.php

```diff
@@ -55,7 +55,7 @@ function AA_add_action_links( $links ) {
 /**
  * Block Initializer.
  */
-require_once plugin_dir_path( __FILE__ ) . 'blocks/src/init.php';
+require_once plugin_dir_path( __FILE__ ) . 'blocks/init.php';
```

Comment:
Moving the init call outside of `src` to make the `register_block_type` call a little easier and to allow you to move to a `render.php` file later